### PR TITLE
chamber: init at 2.8.2

### PR DESCRIPTION
**Pull request by kalekseev**
###### Motivation for this change

[Chamber](https://github.com/segmentio/chamber) is a tool for managing secrets. Currently it does so by storing secrets in SSM Parameter Store, an AWS service for storing secrets.
Basic use case is that you want to allow some IAM role to run service that require configuration, chamber helps manage config params in AWS SSM so iam user can start service by passing config from SSM, eg: `aws-vault exec development -- chamber exec loadbalancers -- nginx`

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4177,6 +4177,12 @@
     githubId = 87115;
     name = "Wael Nasreddine";
   };
+  kalekseev = {
```
Comment (zowoq) [inline maintainers/maintainer-list.nix:4180]: Please move this to a separate commit:
```
maintainers: add kalekseev
chamber: init at 2.8.2
```

**Review by zowoq — COMMENTED** [review]
One nit, otherwise lgtm.

**Comment by kalekseev** [comment]
> One nit, otherwise lgtm.

I made a separate commit for maintainer info, thanks!

**Review by zowoq — APPROVED** [review]
